pyodide-build/pyodide_build/buildall.py

```diff
@@ -327,7 +327,9 @@ def generate_needs_build_set(pkg_map: dict[str, BasePackage]) -> set[str]:
     return needs_build
 
 
-def build_from_graph(pkg_map: dict[str, BasePackage], outputdir: Path, args) -> None:
+def build_from_graph(
+    pkg_map: dict[str, BasePackage], outputdir: Path, args: Any
```

Comment:
```suggestion
    pkg_map: dict[str, BasePackage], outputdir: Path, args: argparse.Namespace
```

Reply:
Pretty sure this breaks tests. But we’ll see.